common/src/logger/loggers/mod.rs

```diff
@@ -1,13 +1,14 @@
 // Copyright (c) 2018-2022 The MobileCoin Foundation
 
-/// Sets chan_size for stdout, gelf, and UDP loggers
-const STDOUT_CHANNEL_SIZE: usize = 100_000;
-const STDERR_CHANNEL_SIZE: usize = 100_000;
-const GELF_CHANNEL_SIZE: usize = 100_000;
-const UDP_CHANNEL_SIZE: usize = 100_000;
```

Comment:
Unifying these into a single const made some DRYing work easier. There's no actual need for them to be separate.